camera_wrappers/CMakeLists.txt

```diff
@@ -49,38 +51,60 @@ endif()
 set( CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/OpenNI2 )
 find_package( OpenNI2 )
 if( OpenNI2_FOUND )
-    include_directories( ${OPENNI2_INCLUDE_DIRS} )
-
+    option( WITH_OPENNI2 "Build support for OpenNI2 SDK." ON )
```

Comment:
The option only exists if OpenNI2 was found? Doesn't it make more sense to always have the option, only check if the option is set, and check fail if it was not found?